jujugui/static/gui/src/app/components/profile/bundle-list/bundle-list.js

```diff
@@ -185,6 +189,8 @@ ProfileBundleList.propTypes = {
     list: PropTypes.func.isRequired,
     url: PropTypes.string.isRequired
   }).isRequired,
+  deployTarget: PropTypes.func.isRequired,
```

Comment:
could those two be grouped in a shape,like modelAPI?

Reply:
Also should deployTarget be renamed to addToCanvas or similar? deployTarget seems a bit ambiguous, what do you think?

Reply:
Good point, that method is used all over the place so I've filed #3388 for a followup.